We are removing social_devel

ronaldtebrake opened the pull request:
We are removing social_devel as per https://github.com/goalgorilla/open_social/pull/2397 and are moving devel as dependency to require-dev of composer.
This also means we can't use this as enabler, however we can enable all submodules directly.

ronaldtebrake committed:
We are removing social_devel

We are removing social_devel as per https://github.com/goalgorilla/open_social/pull/2397 and are moving devel as dependency to require-dev of composer.
This also means we can't use this as enabler, however we can enable all submodules directly.

Ressinel reviewed (approved):
Looks good to me! The new Drush command works as aspected.

ronaldtebrake commented on install/install_script.sh:
```diff
@@ -177,5 +177,5 @@ drush php-eval 'node_access_rebuild()';
 then
   echo "enabling devel modules"
-  drush en social_devel -y
+  drush en -y config_update config_update_ui devel kint devel_generate dblog views_ui field_ui contextual
```
```suggestion
  drush en -y config_update config_update_ui devel devel_generate dblog views_ui field_ui contextual
```

Kint is removed and replaced https://www.drupal.org/project/devel/releases/4.1.1

ronaldtebrake committed:
Update install/install_script.sh

Kint is removed from devel and replaced by kint-php/kint https://www.drupal.org/project/devel/releases/4.1.1
